department-of-veterans-affairs/vets-website
# Add asset transfers

## Summary
Import the 'Asset Transfers' section of the PDF into the digital Income and Assets Statement form (21P-0969)

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86364

## Related issue(s)
n/a

## Associated Pull Request(s)
- https://github.com/department-of-veterans-affairs/vets-json-schema/pull/948

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `income_and_assets_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/income_and_assets_form_enabled
4. Go to `http://localhost:3001/income-and-asset-statement-form-21p-0969` in your browser

## How to verify
1. Continue to the Asset Transfers chapter `/asset-transfers-summary` 
2. Add an asset transfer
3. Verify the asset transfer can be edited and deleted

## What areas of the site does it impact?
Income and Asset Statement Form

## Screenshots
### Summary Page
| No Items | Item Added |
| ----------- | ----------- | 
|![Screenshot 2024-10-31 at 9 53 56 AM](https://github.com/user-attachments/assets/38fc58be-7736-4919-80b7-39784237a2d0)|![Screenshot 2024-10-31 at 10 22 57 AM](https://github.com/user-attachments/assets/1a8991c2-d9ab-4092-a581-ca0a3966753e)

### Item Pages
| Item Page | Item Page | Item Page | Item Page | Item Page | Item Page |
| ----------- | ----------- | ----------- | ----------- |  ----------- | ----------- | 
|![Screenshot 2024-10-31 at 9 54 06 AM](https://github.com/user-attachments/assets/c313fe23-c9ca-4e35-be83-d9f5de5f19b2)|![Screenshot 2024-10-31 at 9 54 11 AM](https://github.com/user-attachments/assets/88cdb213-1a56-499e-bc00-de6a933dfa33)|![Screenshot 2024-10-31 at 9 54 44 AM](https://github.com/user-attachments/assets/632f3711-ee09-40d0-a788-d3196f6d6408)|![Screenshot 2024-10-31 at 9 55 00 AM](https://github.com/user-attachments/assets/2d7b0668-a8d7-48a0-b148-3c15e26edb61)|![Screenshot 2024-10-31 at 9 55 11 AM](https://github.com/user-attachments/assets/af45b3d1-0589-413c-a24d-ffc1a5941052)|![Screenshot 2024-10-31 at 9 55 16 AM](https://github.com/user-attachments/assets/2909dea0-bfe1-4f03-9abd-f7b5c9d1e999)


### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
